### PR TITLE
Use HearthDB.Enums.Rarity

### PR DESCRIPTION
**Pull request by Jecoms**
HDT-Rarity Annotated with Obsolete.

### CODE REVIEW

File: Hearthstone Deck Tracker/Hearthstone/Card.cs [hunk 1]
```diff
@@ -115,7 +114,7 @@ public Card(HearthDb.Card dbCard)
 			Id = dbCard.Id;
 			Count = 1;
 			PlayerClass = HearthDbConverter.ConvertClass(dbCard.Class);
-			Rarity = HearthDbConverter.RariryConverter(dbCard.Rarity);
+             Rarity = dbCard.Rarity;
```
Comment (Jecoms) [inline Hearthstone Deck Tracker/Hearthstone/Card.cs]: I don't know what is wrong with the white space upon submitting a pull request. It looks fine in VS.

Comment (riQQ) [inline Hearthstone Deck Tracker/Hearthstone/Card.cs]: Check `Edit > Advanced > View White Space`. It's a tab vs space issue.

**Comment by Epix37** [comment]
I can squish the commits for you, no problem.

I'm not sure making the old enum obsolete actually solves the problem though, I was waiting on a response from @ko-vasilev. :)

**Comment by ko-vasilev** [comment]
@Epix37 Sure it breaks the plug-in, but it's not hard to modify it to use another type for rarities.

**Comment by Epix37** [comment]
I was hoping there would be a way we could avoid breaking it, but I guess not huh?

**Comment by ko-vasilev** [comment]
I mean, if the Rarity property of Card class stays of the same type and this enum will still be usable it should not break the plug-in. Otherwise I don't think there is a way to keep plug-in compatible.

**Comment by Epix37** [comment]
Hm yeah... I can't really think of a way to make it work other than introducing a new property for the HearthDb Rarity and leaving the old one in as obsolete - which is not a good soluition.

This is not a change I want to break plugin compatibility for I think.

My suggestions now would be to leave this open until I have to break compatibility anyway (if I decide to update MahApps at some point for example).

**Comment by ko-vasilev** [comment]
I mean, it's not a big deal to change the plugin code to use another type for Rarity.

**Comment by Epix37** [comment]
I'm mainly concerned about the user experience. Everyone would have to manually update again.

**Comment by ericBG** [comment]
Maybe it's time to try and make an automatic plugins manager? In fact, that seems like an idea...

**Comment by Epix37** [comment]
might as well merge this now I guess

**Comment by Epix37** [comment]
Merged with https://github.com/HearthSim/Hearthstone-Deck-Tracker/commit/294d737e6482ec897f3d8ba81d3859cff20dfbce.

**Pull request closed without merging**